telemetry/definitions/commonDefinitions.json

```
@@ -773,7 +773,44 @@
             "name": "codecatalyst_devEnvironmentWorkflowStep",
             "type": "string",
             "description": "Workflow step name"
+        },
+        {
+            "name": "authConnectionsCount",
+            "type": "int",
+            "description": "Current number of authentication connections the user has."
```

nit: The majority of definitions don't have a trailing period:
```suggestion
            "description": "Current number of authentication connections the user has"
```

Could shorten "authentication" to "auth" too.